concent_api/utils/tests/test_api_view.py

```diff
@@ -49,9 +60,9 @@ def test_api_view_should_return_golem_message_as_octet_stream(self):
         decoded_message = None
 
         @api_view
-        def dummy_view(request, message):                                       # pylint: disable=unused-argument
+        def dummy_view(request, _message):                                       # pylint: disable=unused-argument
```

Comment:
why `_` added?

Reply:
This is pylint's convention for ignoring warnings about unused parameters: https://pylint.readthedocs.io/en/latest/faq.html#i-have-a-callback-function-where-i-have-no-control-over-received-arguments-how-do-i-avoid-getting-unused-argument-warnings